core/libretro/libretro.cpp

```diff
@@ -353,11 +365,19 @@ void retro_set_environment(retro_environment_t cb)
       },
       {
          "reicast_cable_type",
+#ifdef LOW_END
+         "Cable type; VGA (RGB)|TV (RGB)|TV (Composite)",	
+#else
          "Cable type; TV (RGB)|TV (Composite)|VGA (RGB)",
+#endif
```

Comment:
Does this really impact low-end platforms?
I'm not against changing the default setting (Some games don't like `TV (RGB)`) but I'm trying to limit the conditionals

Reply:
I seemed to have better luck with it but for classic platforms I think this is ok to change.